docs/content/en/docs/reference/clusterspec/packages.md

```diff
@@ -0,0 +1,938 @@
+      </tr></tbody>
+</table>
+
+
+#### Package.status
+<sup><sup>[↩ Parent](#package)</sup></sup>
+
+
+
+PackageStatus defines the observed state of Package.
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+        <td><b>currentVersion</b></td>
+        <td>string</td>
+        <td>
+          Version currently installed.<br/>
+        </td>
+        <td>true</td>
+      </tr><tr>
+        <td><b><a href="#packagestatussource">source</a></b></td>
+        <td>object</td>
+        <td>
+          Source associated with the installation.<br/>
+        </td>
+        <td>true</td>
+      </tr><tr>
+        <td><b>detail</b></td>
+        <td>string</td>
+        <td>
+          Detail of the state.<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b>state</b></td>
+        <td>enum</td>
+        <td>
+          State of the installation.<br/>
+          <br/>
+            <i>Enum</i>: initializing, installing, installed, updating, uninstalling, unknown<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b>targetVersion</b></td>
+        <td>string</td>
+        <td>
+          Version to be installed.<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b><a href="#packagestatusupgradesavailableindex">upgradesAvailable</a></b></td>
+        <td>[]object</td>
+        <td>
+          UpgradesAvailable indicates upgraded versions in the bundle.<br/>
+        </td>
+        <td>false</td>
+      </tr></tbody>
+</table>
+
+
+#### Package.status.source
+<sup><sup>[↩ Parent](#packagestatus)</sup></sup>
+
+
+
+Source associated with the installation.
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+        <td><b>digest</b></td>
+        <td>string</td>
+        <td>
+          Digest is a checksum value identifying the version of the package and its contents.<br/>
+        </td>
+        <td>true</td>
+      </tr><tr>
+        <td><b>registry</b></td>
+        <td>string</td>
+        <td>
+          Registry in which the package is found.<br/>
+        </td>
+        <td>true</td>
+      </tr><tr>
+        <td><b>repository</b></td>
+        <td>string</td>
+        <td>
+          Repository within the Registry where the package is found.<br/>
+        </td>
+        <td>true</td>
+      </tr><tr>
+        <td><b>version</b></td>
+        <td>string</td>
+        <td>
+          Versions of the package supported.<br/>
+        </td>
+        <td>true</td>
+      </tr></tbody>
+</table>
+
+
+#### Package.status.upgradesAvailable[index]
+<sup><sup>[↩ Parent](#packagestatus)</sup></sup>
+
+
+
+PackageAvailableUpgrade details the package's available upgrades' versions.
```

Comment:
Not sure why this says `upgrades'` . Why not just "...the package's available upgrade versions."?

Reply:
This seems to imply that each upgrade could have multiple versions, as opposed to the package having multiple versions.

Reply:
This seems to be a typo. Fixed.